internal/api/v1beta1/app_types.go

```diff
@@ -497,3 +497,18 @@ func (s AppStatus) Condition(t AppConditionType) *AppCondition {
 	}
 	return nil
 }
+
+// PodState describes the simplified state of a pod in the cluster
+type PodState string
```

Comment:
I would not introduce another pod state type, even internally. It would be confusing.

Reply:
Well, yes, but as this is not a direct translation of apiv1.PodPhase, and we have this overridden in some cases - seems that we don't have another choice than having separate states.